networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc

```diff
@@ -9,6 +9,11 @@ toc::[]
 
 You can update the node network configuration, such as adding or removing interfaces from nodes, by applying `NodeNetworkConfigurationPolicy` manifests to the cluster.
 
+[NOTE]
+====
+The Kubernetes NMState Operator updates the network configuration of the secondary NIC. It cannot update the network configuration of the primary NIC.
+====
+
 include::modules/virt-about-nmstate.adoc[leveloffset=+1]
```

Comment:
Isn't this still going to result in basically a duplicate note? I think we just want to include the virt-about-nmstate module anywhere we want to document this.

Reply:
Looking at the complete section there seems to be a lot of duplication I am new to this Operator but @cybertron is it true to say we need Kubernetes NMState Operator to do all the stuff that follows **Managing policy from the web console**

Reply:
Yes, the console is just an interface to the operator. It won't do anything (and shouldn't show up) if the operator is not installed.